### ISSUE TITLE
Support for AV1 codec for images and videos

### ISSUE THREAD

**Issue by dillfrescott** [jeremyckahn/chitchatter#106]
Images use the `.avif` format, and right now they show up as just [this](https://user-images.githubusercontent.com/56714680/224505911-429babd6-b675-4943-bb81-aa35d0a2b7bb.png).

Having the ability to display these types of files would be immensely helpful!

**Comment by jeremyckahn** [comment]
Good idea @dillfrescott! To achieve this, I believe we'd need to make a PR to add `.avif` to [this list](https://github.com/feross/render-media/blob/a445b2ab90fcd4a248552d32027b2bc6a02600c8/index.js#L55-L62) in https://github.com/feross/render-media.
